### PR TITLE
New order tabs

### PR DESCRIPTION
**Pull request by AWare**

### CODE REVIEW

File: assets/javascripts/src/components/pages/Contribution.jsx [hunk 1]
```diff
@@ -18,23 +18,14 @@ export default class Contribution extends React.Component {
...
-    }
-
     handleClick(amount) {
         this.setState({
             inputAmount: '',
             highlightButton: true
         });
 
         this.props.setAmount(amount);
+        console.log('clickj');
```
Comment (alexduf) [inline assets/javascripts/src/components/pages/Contribution.jsx]: you probably want to remove that

**Review by alexduf — APPROVED** [review]
:+1:

File: assets/javascripts/src/components/pages/Contribution.jsx (hunk 1 above)
Comment (AWare) [reply assets/javascripts/src/components/pages/Contribution.jsx]: Yeah, I've got some css changes from Ben too.

File: assets/javascripts/src/components/pages/Payment.jsx [hunk 2]
```diff
@@ -6,7 +6,8 @@ import InputField from '../InputField';
 import CardIcon from '../CardIcon';
 
 export default class Payment extends React.Component {
-    formatExpiry(expiry) {
+
+       formatExpiry(expiry) {
```
Comment (desbo) [inline assets/javascripts/src/components/pages/Payment.jsx]: lint error ;)

**Comment by AWare** [comment]
image

Using Ben's new focus/hover state*. The old one was really non-obvious, which wasn't great for tabbing. Considering making this a focus state only.

*Ben actually wanted an outer border, not an inner one.

**Review by desbo — APPROVED** [review]
👍

**Review by jranks123 — APPROVED** [review]
👍

**Pull request merged**